extensions/http-curl/processors/InvokeHTTP.cpp

```diff
@@ -33,6 +33,7 @@
 #include <utility>
 #include <vector>
 
+#include "../../../libminifi/include/utils/ByteArrayCallback.h"
```

Comment:
Awkward include path. Perhaps include path could be set better in CMakeLists.txt?

Reply:
cmake lists? huh?

no, that's eclipse. it does that to me. thanks!